introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello! My name is Mike Robards, and I am Mastering GitHub with Code School!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.